### PR TITLE
Update DatePicker to be more responsive on Larger screens.

### PR DESCRIPTION
**Pull request by ssancho14**
#### WHAT

Update DatePicker to be more responsive on Larger screens

#### WHY

More variety of devices in the market.

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

File: composables/src/main/java/com/google/android/horologist/composables/DatePicker.kt [hunk 1]
```diff
@@ -107,8 +112,16 @@ public fun DatePicker(
         rememberPickerGroupState(FocusableElementDatePicker.DAY.index)
     }
 
+    val breakpoint = LocalConfiguration.current.screenWidthDp > 225
```
Comment (yschimke) [inline composables/src/main/java/com/google/android/horologist/composables/DatePicker.kt:114]: can we call this isLarge?

File: sample/src/main/java/com/google/android/horologist/sample/SampleWearApp.kt [hunk 2]
```diff
@@ -76,6 +96,7 @@ fun SampleWearApp() {
     val navController = rememberSwipeDismissableNavController()
 
     var time by remember { mutableStateOf(LocalDateTime.now()) }
+    val d0 = LocalDensity.current
```
Comment (yschimke) [inline sample/src/main/java/com/google/android/horologist/sample/SampleWearApp.kt:80]: ?

File: sample/src/main/java/com/google/android/horologist/sample/MenuScreen.kt [hunk 3]
```diff
@@ -25,6 +25,7 @@ import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
```
Comment (yschimke) [inline sample/src/main/java/com/google/android/horologist/sample/MenuScreen.kt]: revert?

File: compose-layout/src/main/java/com/google/android/horologist/compose/layout/AppScaffold.kt [hunk 4]
```diff
@@ -57,4 +90,136 @@ fun AppScaffold(
...
+                ) {
+                    Text("Round")
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun ToggleRow(
```
Comment (yschimke) [inline compose-layout/src/main/java/com/google/android/horologist/compose/layout/AppScaffold.kt:190]: We should discuss another way to achieve this. Potentially a different XScaffold that plays around with the same composition locals?

File: composables/src/main/java/com/google/android/horologist/composables/DatePicker.kt (hunk 1 above)
Comment (ssancho14) [reply composables/src/main/java/com/google/android/horologist/composables/DatePicker.kt:114]: Done

File: sample/src/main/java/com/google/android/horologist/sample/SampleWearApp.kt (hunk 2 above)
Comment (ssancho14) [reply sample/src/main/java/com/google/android/horologist/sample/SampleWearApp.kt:80]: Removed.

File: sample/src/main/java/com/google/android/horologist/sample/MenuScreen.kt (hunk 3 above)
Comment (ssancho14) [reply sample/src/main/java/com/google/android/horologist/sample/MenuScreen.kt]: Done.

File: compose-layout/src/main/java/com/google/android/horologist/compose/layout/AppScaffold.kt (hunk 4 above)
Comment (ssancho14) [reply compose-layout/src/main/java/com/google/android/horologist/compose/layout/AppScaffold.kt:190]: Removed for now, forgot it was already there on the samples/.../scratch/DialogActivity. 
We can use that for debugging.

**Review by BowerSteve — APPROVED** [review]
Looks good to me - also need updates to the TimePicker, will they be in a separate PR?

**Pull request merged**